src/features/colinks/NotFound.tsx

```diff
@@ -0,0 +1,10 @@
+import { Flex, Text } from 'ui';
+
+export const NotFound = () => {
```

Comment:
I think this should just be in src/pages/colinks/NotFound its not a feature. @sshmm